docs/docs/welcome.mdx

```diff
@@ -0,0 +1,90 @@
+---
+slug: /
+title: Welcome to Nebari's documentation
+---
+
+export const invisible_table = {
+   color: 'blue',
+   lineHeight: 10,
+   padding: '1.5em',
+   background: 'white',
+   backgroundColor: 'white',
+}
+
+:::warning We are currently undergoing a rename from [QHub](https://docs.qhub.dev/) to Nebari.   
+:construction: Please excuse our dust as we build out our new docs :) :construction:
+:::
+
+[INSERT NEBARI LOGO HERE]
+
+Nebari: The Roots of a Bonsai Tree
+
+## Nebari Vision
+
+An opinionated open source [JupyterHub][jupyterhub] distribution that provides 
+an all-in-one development and experimentation platform for teams to work 
+efficiently and collaboratively.
+
+Deployment, integration and maintenance is minimized by using the 
+infrastructure as code (IaC) management approach.
+
+## Nebari has a *DevOps for non-DevOps people* approach.
+
+![Nebari overview sequence](/img/nebari_overview_sequence.png)
+
+### Components
+
+The technology stack is an integration of the following existing open source libraries:
+
+- [**Terraform**](https://www.terraform.io/intro/index.html): a tool for building, changing, and versioning infrastructure.
```

Comment:
Is the **bold** text necessary here? all these will be links so already distinctive from the surrounding text